Sources/HttpRouter.swift

```diff
@@ -106,17 +109,17 @@ open class HttpRouter {
     ///   - matchedNodes: An array with the nodes matching the route
     ///   - index: The index of current position in the generator
     ///   - count: The number of elements if the route to match
-    private func findHandler(_ node: inout Node, params: inout [String: String], generator: inout IndexingIterator<[String]>, matchedNodes: inout [Node], index: Int, count: Int) {
+    private func findHandler(_ node: inout Node, params: inout [String: String], pattern: [String], matchedNodes: inout [Node], index: Int, count: Int) {
```

Comment:
The swift doc on this method still references generator and indexing iterator

Reply:
Uhmm I forgot to update it. Let's fix it 👍.